src/components/modes/teacher/BackgroundToggle.js

```diff
@@ -25,26 +25,21 @@ const useStyles = makeStyles(() => ({
 const BackgroundToggle = () => {
   const classes = useStyles();
   const { t } = useTranslation();
-  const dispatch = useDispatch();
 
-  const { backgroundImage } = useSelector(
-    ({ appInstance }) => appInstance.content.settings,
-  );
+  const { mutate: patchSettings } = useMutation(MUTATION_KEYS.PATCH_SETTINGS);
+  const context = useContext(Context);
+  const backgroundImage = context?.get('settings')?.backgroundImage;
```

Comment:
I think this have to change. We use app-settings now, I remove `PATCH_SETTINGS` completely. Instead you will use GET / POST / PATCH / DELETE app-settings

Reply:
Fixed!